website/docs/reference/actions.mdx

```diff
@@ -0,0 +1,72 @@
+---
+title: Actions
+---
+
+:::info Availability
+
+Unleash actions is an **enterprise-only** feature released as beta in Unleash v5.11.
+
+:::
+## Overview
+
+Actions enable our users to execute automated actions based on signals received from an external system.
+
+![Actions executed by Unleash.](/img/signals/action-execution.png)
+
+## Creating actions
+
+:::info Permissions
+
+During the beta phase, creating actions require admin permissions.
+
+:::
+
+Actions are scoped to a project. To be able to create actions, you need to navigate to the project settings and find the **Actions** tab.
+
+![Actions tab.](/img/signals/actions-tab.png)
+
+### Name and description
+When creating a new action you first need to specify a name and a description. This will help you identify your action and give it a meaningful description.
+
+### Source and filters
+
+Then you need to select a source. If you haven't created a signal endpoint yet, you can do so by clicking the **Create new signal endpoint** button (more information on this in the [Signals](./signals.mdx) section).
+
+Once selected the source, you'll be able to see a preview of the latest signal which will help you to define different filters that need to match before executing the action.
+
+Filters work similar as [feature flag strategy constraints](./strategy-constraints.md) where the signal payload acts as the context for the constraint evaluation.
+
+### Actions
+
+When Unleash executes actions as reaction to an external signal, it will use a [service account](./service-accounts.md) to execute them. This service account needs to have the necessary permissions to execute the actions. This service account doesn't need an API token, so when creating it, don't generate an API token and give it the necessary permissions only.
+
+The actions are executed in the order they are defined. You can add multiple actions to an action set. Depending on the action you want to execute you will have to provide different parameters.
+
```

Comment:
Maybe these sections could use some screenshots.

Reply:
Addressed in https://github.com/Unleash/unleash/pull/6641/commits/fc401d2e0ba8cd3e2f9aae103eea68f064ae4d85